repositories/ingested_data_indexes_repository.go

```diff
@@ -186,14 +177,16 @@ func createIndexSQL(ctx context.Context, exec Executor, index models.ConcreteInd
 			),
 		)
 	}
+	sql += "WHERE valid_until='infinity'"
```

Comment:
I didn't know a `where` close could be added to an index 🎉

Reply:
yes, it allows to reduce the cpu/storage size of the index depending on how you're using it :-) 
it's actually quite powerful